mne/io/tests/test_meas_info.py

```diff
@@ -199,13 +201,27 @@ def test_read_write_info():
 
     write_info(temp_file, info)
     info = read_info(temp_file)
-    assert_equal(info['proc_history'][0]['creator'], creator)
-    assert_equal(info['hpi_meas'][0]['creator'], creator)
-    assert_equal(info['subject_info']['his_id'], creator)
-    assert_equal(info['gantry_angle'], gantry_angle)
+    assert info['proc_history'][0]['creator'] == creator
+    assert info['hpi_meas'][0]['creator'] == creator
+    assert info['subject_info']['his_id'] == creator
+    assert info['gantry_angle'] == gantry_angle
     assert info['subject_info']['height'] == 2.3
     assert info['subject_info']['weight'] == 11.1
 
+    # Test that writing twice produces the same file
+    m1 = hashlib.md5()
+    with open(temp_file, 'rb') as fid:
+        m1.update(fid.read())
+    m1 = m1.hexdigest()
+    temp_file_2 = op.join(tempdir, 'info2.fif')
+    assert temp_file_2 != temp_file
+    write_info(temp_file_2, info)
+    m2 = hashlib.md5()
+    with open(temp_file_2, 'rb') as fid:
+        m2.update(fid.read())
+    m2 = m2.hexdigest()
+    assert m1 == m2
```

Comment:
These changes are new (should fail on `master` because we used `date_now` when writing, so this is lost information but arguably a good choice?)